distribution/sdk/sample-mgmp-server/src/main/resources/mock-mgmp/__files/MgmpGetRecordsResponse.xml

```diff
@@ -0,0 +1,640 @@
+									</gmd:valueType>
+									<gmd:valueUnit xlink:href="http://www.opengis.net/def/uom/OGC/1.0/metre"/>
+									<gmd:value>
+										<gco:Record xsi:type="xs:double" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">159</gco:Record>
+									</gmd:value>
+								</gmd:DQ_QuantitativeResult>
+							</gmd:result>
+						</gmd:DQ_AbsoluteExternalPositionalAccuracy>
+					</gmd:report>
+					<gmd:report>
+						<gmd:DQ_RelativeInternalPositionalAccuracy>
+							<gmd:nameOfMeasure>
+								<gco:CharacterString>relative horizontal error</gco:CharacterString>
+							</gmd:nameOfMeasure>
+							<gmd:nameOfMeasure>
+								<gco:CharacterString>Rel CE90</gco:CharacterString>
+							</gmd:nameOfMeasure>
+							<gmd:measureIdentification>
+								<gmd:RS_Identifier>
+									<gmd:code>
+										<gco:CharacterString>53</gco:CharacterString>
+									</gmd:code>
+									<gmd:codeSpace>
+										<gco:CharacterString>ISO19157:2013</gco:CharacterString>
+									</gmd:codeSpace>
+								</gmd:RS_Identifier>
+							</gmd:measureIdentification>
+							<gmd:measureDescription>
+								<gco:CharacterString>evaluation of the random errors in the horizontal position of one feature to another in the same dataset or on the same map/chart</gco:CharacterString>
+							</gmd:measureDescription>
+							<gmd:result gco:nilReason="missing"/>
+						</gmd:DQ_RelativeInternalPositionalAccuracy>
+					</gmd:report>
+					<gmd:report>
+						<gmd:DQ_RelativeInternalPositionalAccuracy>
+							<gmd:nameOfMeasure>
+								<gco:CharacterString>relative vertical error</gco:CharacterString>
+							</gmd:nameOfMeasure>
+							<gmd:nameOfMeasure>
+								<gco:CharacterString>Rel LE90</gco:CharacterString>
+							</gmd:nameOfMeasure>
+							<gmd:measureIdentification>
+								<gmd:RS_Identifier>
+									<gmd:code>
+										<gco:CharacterString>52</gco:CharacterString>
+									</gmd:code>
+									<gmd:codeSpace>
+										<gco:CharacterString>ISO19157:2013</gco:CharacterString>
+									</gmd:codeSpace>
+								</gmd:RS_Identifier>
+							</gmd:measureIdentification>
+							<gmd:measureDescription>
+								<gco:CharacterString>evaluation of the random errors of one relief feature to another in the same dataset or on the same map/chart. It is a function of the random errors in the two elevations with respect to a common vertical datum</gco:CharacterString>
+							</gmd:measureDescription>
+							<gmd:result gco:nilReason="missing"/>
+						</gmd:DQ_RelativeInternalPositionalAccuracy>
+					</gmd:report>
+				</gmd:DQ_DataQuality>
+			</gmd:dataQualityInfo>
+			<gmd:metadataConstraints>
+				<mgmp:MGMP_SecurityConstraints>
+					<gmd:classification>
+						<gmd:MD_ClassificationCode codeList="http://mod.uk/spatial/codelist/mgmp/2.0/MGMP_ClassificationUK_GovCode" codeListValue="official">Official</gmd:MD_ClassificationCode>
```

Comment:
Please either change this or make sure there is a comment stating this is test data and there is no classification.